### PR TITLE
Adapter updates for App open ad formate.

### PR DESCRIPTION
**Pull request by ManojBudumuru-vungle**
This commit will add support for AppOpen ad in the adapter.

IOS-5551

### CODE REVIEW

File: Vungle/VungleAdapter/ALVungleMediationAdapter.h [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface ALVungleMediationAdapter : ALMediationAdapter<MASignalProvider, MAInterstitialAdapter, MARewardedAdapter, MAAdViewAdapter, MANativeAdAdapter>
+@interface ALVungleMediationAdapter : ALMediationAdapter<MASignalProvider, MAInterstitialAdapter, /* MAAppOpenAdapter */ MARewardedAdapter, MAAdViewAdapter, MANativeAdAdapter>
```
Comment (stanley-vungle) [inline Vungle/VungleAdapter/ALVungleMediationAdapter.h:13]: Why's the new adapter commented out? Add a comma too

File: Vungle/VungleAdapter/ALVungleMediationAdapter.m [hunk 2]
```diff
@@ -477,6 +562,65 @@ - (void)interstitialAdDidClick:(VungleInterstitial * _Nonnull)interstitial
...
+    }
+    else
+    {
+        [self.appOpenAdDelegate didDisplayAppOpenAd];
+    }
+}
+
+- (void)interstitialAdDidFailToPresent:(VungleInterstitial * _Nonnull)interstitial withError:(NSError * _Nonnull)withError
+{
+    MAAdapterError *adapterError = [MAAdapterError errorWithCode: kALErrorCode errorString: @"Ad Display Failed" mediatedNetworkErrorCode: withError.code mediatedNetworkErrorMessage: withError.localizedDescription];;
```
Comment (stanley-vungle) [inline Vungle/VungleAdapter/ALVungleMediationAdapter.m]: extra ; at the end here

**Review by stanley-vungle — APPROVED** [review]
LGTM

File: Vungle/VungleAdapter/ALVungleMediationAdapter.h (hunk 1 above)
Comment (YueVungle) [reply Vungle/VungleAdapter/ALVungleMediationAdapter.h:13]: Have the same question, might need to add `// TODO:` here.

Comment (ManojBudumuru-vungle) [reply Vungle/VungleAdapter/ALVungleMediationAdapter.h:13]: @YueVungle will update it once 6.12 is updated so. no todos here. 👍

**Pull request merged**